src/main/app/case/case-api.ts

```diff
@@ -59,9 +60,14 @@ export class CaseApi {
     return this.getPriorityUserCase(userCases);
   }
 
+  public async getUsersRoleOnCase(caseId: string, userId: string): Promise<UserRole> {
```

Comment:
maybe move this below isApplicant2 so that it appears after its usage

Reply:
Done.